lib/jekyll.py

```diff
@@ -91,6 +95,13 @@ def build_website(json_root, md_root, site_url, html_root, title, zulip_url, zul
                 date_footer,
                 )
 
+# writes the _config.yml file, currently only needed for last_updated
+# `streams`: a dict mapping stream names to stream json objects as described in the header.
+def write_config(md_root, streams):
+    last_updated = format_date1(stream_info['time'])
```

Comment:
I ran your script, and it has an error in this line
```
archive.py 192 <module>
run()

archive.py 181 run
build_website(

jekyll.py 64 build_website
write_config(md_root, stream_info)

jekyll.py 101 write_config
last_updated = format_date1(stream_info['time'])

NameError:
name 'stream_info' is not defined
```

Let me know if you don't have the time to update this PR, then I can find someone else to take over.

Reply:
Thanks for the review. I'll have a look 🙂